src/control/cam_control_mod.meta

```diff
@@ -0,0 +1,32 @@
+[ccpp-table-properties]
+  name = cam_control_mod
+  type = module
+
+[ccpp-arg-table]
+  name  = cam_control_mod
+  type  = module
+[ aqua_planet ]
+  standard_name = do_aqua_planet
+  units = 1
+  type = logical
+  dimensions = ()
+[ eccen ]
+  standard_name = eccentricity_factor
+  units = 1
+  type = real | kind = r8
+  dimensions = ()
+[ obliqr ]
+  standard_name = obliquity
```

Comment:
I might make this standard name a little more explicit, and add a long name:

```suggestion
  standard_name = planet_obliquity
  long_name = planet's axial tilt (obliquity)
```

Reply:
changed & added!